### PR TITLE
:tada: initial impl. for #131

### PR DESCRIPTION
**Pull request by Dev-CasperTheGhost**
image

## Bug

- [ ] Related issues linked using `closes: #number`

## Feature

- [x] Related issues linked using closes: #131 
- [ ] There is only 1 db migration with no breaking changes.

## Translations

- [ ] `.json` files are formatted: `yarn format`
- [ ] Translations are correct
- [ ] New translation? It's been added to `next.config.js`

### CODE REVIEW

**Comment by pp2amd6** [comment]
> Starting on this; still lots of work to do!

I have already come up with many improvements)))

**Comment by Dev-CasperTheGhost** [comment]
> > Starting on this; still lots of work to do!
> 
> I have already come up with many improvements)))

Feel free to comment them here!

File: packages/client/src/components/leo/modals/CreateTicketModal.tsx [hunk 1]
```diff
@@ -143,3 +150,40 @@ export const CreateTicketModal = ({ type }: { type: RecordType }) => {
...
+      <table className="w-full overflow-hidden whitespace-nowrap max-h-64">
+        <thead>
+          <tr>
+            <th>{"Penal Code"}</th>
+            <th>{"Data"}</th>
+          </tr>
+        </thead>
+        <tbody>
+          {penalCodes.map((penalCode) => (
+            // todo: <TableItem penalCode={penalCode}  /> -> keep track of state in this component
```
Comment (Dev-CasperTheGhost) [inline packages/client/src/components/leo/modals/CreateTicketModal.tsx:170]: todo:

**Comment by Dev-CasperTheGhost** [comment]
Closed by #197

**Pull request closed without merging**